docs/maps/maps-getting-started.asciidoc

```diff
@@ -1,9 +1,9 @@
 [role="xpack"]
 [[maps-getting-started]]
-== Create a map with multiple layers and data sources
+== Build, analyze, and share a map to compare metrics by country or region
```

Comment:
The word "share" in Kibana has specific meaning in Kibana tied to reporting, embedding, and creating URLs. See below for the UI in Dashboard. This tutorial does not cover sharing in this traditional sense. How about "Build, analyze, and add a map to a dashboard to compare metrics by country or region"?

<img width="300" alt="Screen Shot 2021-01-28 at 8 33 02 AM" src="https://user-images.githubusercontent.com/373691/106160934-9bd40a80-6143-11eb-929d-42a1ab99610b.png">

Reply:
"Analyze" and "compare" seem the same to me.  What if the title were a little simpler:

Build a map to compare metrics by country or region